### ISSUE TITLE
Add ConnectionString Property in target

### ISSUE THREAD

**Issue by RavindraBarapartre** [harouny/NLog.Extensions.AzureTableStorage#19]
Hello,
In my application I am not allowed to store storage Connection String in Config. Will you please add connection string property in target.

Thanks

**Comment by harouny** [comment]
Hi @RavindraBarapartre 
What type of configuration file you are using? Can you add an app settings key/value or a Cloud Service configuration file entry?

**Comment by RavindraBarapartre** [comment]
We are storing config in database.
we want to assign these property run time.

**Comment by JoelGallagher** [comment]
Assigning config at runtime would be fantastic for my project ++++

**Comment by caseyjmorris** [comment]
I agree this would be kind of nice; this guy has worked it into his fork but I think you'd need a little more finesse to support both scenarios:  https://github.com/abkonsta/NLog.Extensions.AzureTableStorage

**Comment by RavindraBarapatre** [comment]
Thanks for fix.
This will really help us.
